vpr/src/place/place.cpp

```diff
@@ -2590,15 +2592,13 @@ static void alloc_and_load_for_fast_cost_update(float place_cost_exp) {
      * subhigh must be greater than or equal to sublow, we only need to       *
      * allocate storage for the lower half of a matrix.                       */
 
-    chanx_place_cost_fac = (float**)vtr::malloc(
-        (device_ctx.grid.height()) * sizeof(float*));
+    chanx_place_cost_fac = new float*[(device_ctx.grid.height())];
```

Comment:
Might be able to change these to a vtr_matrix (I think you can -- they are a 2D array, so a vtr_matrix should be a good replacement). Can leave for a future PR if you wish, but good to file an issue on and schedule a clean up.

Reply:
See the vtr utilities documentation on the vtr website for information on vtr_matrix.